cluster/images/kubelet/Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM centos
-ADD kubelet /kubelet
+FROM debian:jessie
```

Comment:
We also need to support `--containerized` flag which runs `nsenter`. However I don't know what's the proper image to use here. I'll leave a TODO in Makefile to figure it out.